src/pages/edge/edge-network/tutorials/send-datastream-overrides.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Sending datastream overrides to Edge Network
+description: Learn how to 
+keywords:
+- Edge Network
+- Datastream config overrides
+- Datastream ID override
+- Guide
+---
+
+# Sending datastream overrides to Edge Network
+
+Datastream overrides let you define additional configuration for your datastreams, allowing you to trigger different datastream behaviors without either creating a new datastream or modifying your existing settings.
+
+## Prerequisites
+
+Before continuing with this tutorial, read the following guides:
```

Comment:
can we remove this sentence or rephrase since the first point below is about minimum extension version required and not really a guide